config/main.py

```diff
@@ -544,7 +546,7 @@ def reload(filename, yes, load_sysinfo):
         command = "{} -H -k {} --write-to-db".format(SONIC_CFGGEN_PATH, cfg_hwsku)
         run_command(command, display_cmd=True)
 
-    command = "{} -j {} --write-to-db".format(SONIC_CFGGEN_PATH, filename)
+    command = "{} -j {} -j {} --write-to-db".format(SONIC_CFGGEN_PATH, filename, init_cfg_file)
```

Comment:
Here, you should check for the presence of the init_cfg.json file and run the appropriate command based on whether or not it exists. See how it's done in the load_minigraph command on lines 609-612.

Reply:
Added a if statement to decide whether the file init_cfg.json exists or not.